ui/src/cloud/apis/demodata.ts

```diff
@@ -106,3 +99,30 @@ export const fetchDemoDataBuckets = async (): Promise<Bucket[]> => {
     return [] // demodata bucket fetching errors should not effect regular bucket fetching
   }
 }
+
+export const getNormalizedDemoDataBucket = async (
+  bucketID: string
+): Promise<NormalizedSchema<BucketEntities, string>> => {
+  const resp = await getBucket({bucketID})
```

Comment:
just a sanity check: we don't need to wrap this in a `try / catch` do we ?

Reply:
it's called in a function that try/catch'es, and I want that function to handle the error conditions. I would like to write a try/catch which throws the error up, but apparently that is against our linter rules. It does make me worry that someone will use this function without properly try/catching but... yeah.